docs/administration/snapshot-management.md

```diff
@@ -24,10 +24,14 @@ Starting with Kura 6.0, it is possible to download _a partial_ version of the sn
 
 In the snapshot download popup, a list of checkboxes is shown, representing the list of all the service configurations that can be downloaded. By default, all configurations (represented by the component PIDs) are selected. The user can remove from the downloaded snapshot the configuration they're not interested in by deselecting the related PIDs checkboxes. The snapshot can be then downloaded in XML or JSON format thorough the dedicated button.
 
+Through the search box above the list, it is possible to easily find the desired services configuration, filtering them by their PIDs.
```

Comment:
A search box is available for the user to simplify the process of PID selection for the configurations available in the selected snapshot. As an example, in the screenshot below, it can be seen the filtering in progress as the user writes its filtering keyword